src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,66 @@
+.contacts {
+
+  &__content {
+    @include pageGrid;
+  }
+
+  &__form {
+    display: grid;
+    row-gap: 16px;
+
+    grid-column: 1 / -1;
+
+    @include onTablets {
+      grid-column: 1 / 4;
+    }
+
+    @include onDesktop {
+      grid-column: 1 / 6;
+    }
+  }
+
+  &__button {
+    margin-top: 16px;
+  }
+
+  &__information {
+    display: grid;
+    row-gap: 24px;
+    padding-block: 48px;
+
+    grid-column: 1 / -1;
+
+    @include onTablets {
+      grid-column: 4 / 7;
+      padding-block: 0 100px;
+    }
+
+    @include onDesktop {
+      grid-column: 8 / 11;
+    }
+  }
+
+  &__link {
+    color: $main-text-color;
+    text-decoration: none;
+    font-style: normal;
+    font-weight: 500;
```

Comment:
Normal and 500 are default values, you can remove these everywhere. Need to fix it everywhere
```suggestion
```